<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAnamnesis</class>
 <widget class="QDialog" name="NewAnamnesis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anamnesis</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_patient">
    <attribute name="title">
     <string>Identification</string>
    </attribute>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_sex">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sex</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_dob">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Date of Birth</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ID">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_id">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_sex">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_name_height">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_name_weight">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_height">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>66</width>
       <height>31</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_weight">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>66</width>
       <height>31</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_name_heightUnit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_name_weightUnit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>kg</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Patient...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_notes">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_notes">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Notes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_cc">
    <attribute name="title">
     <string>CC/HPI/PMH</string>
    </attribute>
    <widget class="QLabel" name="label_chiefComplaint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Chief Complaint (CC)</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_chiefComplaint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_historyPresentIllness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_historyPresentIllness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>History of the Present Illness (HPI)</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_pastMedicalHistory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_pastMedicalHistory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Past Medical History (PMH)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_ros">
    <attribute name="title">
     <string>ROS</string>
    </attribute>
    <widget class="QLabel" name="label_cardiovascular">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cardiovascular System</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_cardiovascular">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_respiratory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_respiratory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Respiratory System</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_gastrointestinal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_gastrointestinal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gastrointestinal System</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_genitourinary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_genitourinary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Genitourinary System</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_nervous">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_nervous">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nervous System</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_cranial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_cranial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cranial Nerves Symptoms</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_endocrine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_endocrine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Endocrine System</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_musculoskeletal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_musculoskeletal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Musculoskeletal System</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_skin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_skin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Skin</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_other1">
    <attribute name="title">
     <string>Other Info 1</string>
    </attribute>
    <widget class="QLabel" name="label_family">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Family Diseases</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_family">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_childhood">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_childhood">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Childhood Diseases</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_social">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Social History</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_social">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_other2">
    <attribute name="title">
     <string>Other Info 2</string>
    </attribute>
    <widget class="QLabel" name="label_medications">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Medications</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_medications">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_sexual">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sexual History</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_sexual">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_allergies">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Allergies</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_allergies">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="conclusion">
    <attribute name="title">
     <string>Conclusion</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit_conclusion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>511</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_conclusion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Conclusion</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>590</y>
     <width>166</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
